src/realm/sync/client.cpp

```diff
@@ -1983,35 +1983,41 @@ void SessionWrapper::handle_pending_client_reset_acknowledgement()
 
     auto pending_reset = _impl::client_reset::has_pending_reset(*m_db->start_frozen());
     REALM_ASSERT(pending_reset);
-    m_sess->logger.info("Tracking pending client reset of type \"%1\" from %2", pending_reset->type,
-                        pending_reset->time);
+    m_sess->logger.info(util::LogCategory::reset, "Tracking pending %1", pending_reset->to_string());
+    if (pending_reset->error) {
+        m_sess->logger.info(util::LogCategory::reset, "Originating client reset error: %1", *pending_reset->error);
+    }
+    // Now that the client reset merge is complete, wait for the changes to synchronize with the server
     async_wait_for(true, true, [self = util::bind_ptr(this), pending_reset = *pending_reset](Status status) {
         if (status == ErrorCodes::OperationAborted) {
             return;
         }
         auto& logger = self->m_sess->logger;
         if (!status.is_ok()) {
-            logger.error("Error while tracking client reset acknowledgement: %1", status);
+            logger.error(util::LogCategory::reset, "Error while tracking client reset acknowledgement: %1", status);
             return;
         }
 
+        logger.debug(util::LogCategory::reset, "%1 has been acknowledged by the server. ", pending_reset.to_string());
+        if (pending_reset.error) {
```

Comment:
do we want to include the error in the output of to_string rather than logging on a separate line?